### DISCUSSION TITLE
Config Option to forward login to /auth/oidc instead of the normal login screen.

### DISCUSSION THREAD

**Discussion by minozheros** [outline/outline#5962] [Ideas]
Would it be possible to add a config option, so the login screen could be changed from the normal login screen to /auth/oidc. 
That way, if using oidc auth, the login would happen automatically after the oidc provider authorises the user or it would at least start the login process automatically. It is possible to redirect the page, but it would be more convenient, if you could just set a option withint the environment for it to happen.

**Comment by tommoor** [comment]
Wondering if an option is necessary, or if this could be the default behavior when only a single authentication provider is configured 🤔

**Comment by microbearlogist** [comment]
I would support this as default behavior with single authentication provider.
As the actual behavior only adds one "useless" step for the user (since he do not have any choice), it would be more user friendly.

**Comment by helmut72** [comment]
This behavior would be great if there is only one login option set.

**Comment by helmut72** [comment]
Yeah, this works now. Fantastic.

But... ;) ... [This should be implemented soon](https://github.com/outline/outline/issues/3954)

User was never logged out before, but now it's just a reload of the page when he click to logout.
